send_test.go

```diff
@@ -105,8 +106,92 @@ func TestSQS_SendMessage(t *testing.T) {
 }
 
 func TestSQS_SendMessageWithContext(t *testing.T) {
-	assert.PanicsWithValue(t, "SendMessageWithContext is not implemented", func() {
```

Comment:
@elliotchance, these tests are pretty much a copy'n'paste from the tests from SendMessage(). As in the code is largely duplicated. Not sure if that is a great idea or not?

Reply:
Copying tests is totally fine. It just so happens they are testing the same logic right now.